src/main/java/uk/ac/ebi/embl/converter/gff3/GFF3Annotation.java

```diff
@@ -99,4 +103,18 @@ public void writeGFF3String(Writer writer) throws WriteException {
     public void addFeature(GFF3Feature feature) {
         features.add(feature);
     }
+
+    public void merge(GFF3Annotation other) {
+        this.features.addAll(other.features);
+    }
+
+    public String getAccession() {
+        Optional<GFF3SequenceRegion> directive = Optional.ofNullable(this.sequenceRegion);
+        return directive
+                .map((d) -> d.accession())
+                .orElse(this.features.stream()
+                        .findFirst()
+                        .map(GFF3Feature::accession)
+                        .orElse(null));
```

Comment:
I the accession is null, then there is something wrong, It should be error i think